src/layouts/Landing/index.jsx

```diff
@@ -17,36 +18,28 @@ export const LandingLayout = ({ children, meta = {} }) => {
   const productDomain = 'optrispace.com'
   const productTitle = 'OptriSpace'
   const pageTitle = title ? `${title} | ${productTitle}` : productTitle
-
   const defaultDescription =
     'OptriSpace brings together digital and IT experts, founders, freelancers, stakeholders and enthusiasts ' +
     'and provides a decentralized platform for collaboration and services exchange'
 
+  const pageDescription = description
+    ? description + ' - ' + defaultDescription
+    : defaultDescription
+
   const { domain } = publicRuntimeConfig
 
   return (
     <>
       <Head>
-        <title>{pageTitle}</title>
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta name="description" content={description || defaultDescription} />
-
-        <meta property="og:title" content={title} />
-        <meta property="og:description" content={description} />
-        <meta property="og:url" content={`https://${productDomain}`} />
-        <meta property="og:type" content="website" />
-
-        <meta name="twitter:card" content="summary_large_image" />
-        <meta name="twitter:site" content="@optrispace" />
-        <meta property="twitter:domain" content={productDomain} />
-        <meta property="twitter:url" content={`https://${productDomain}`} />
-        <meta name="twitter:title" content={title} />
-        <meta name="twitter:description" content={description} />
-
         <Favicon />
       </Head>
 
       <Container>
+        <SEOTags
```

Comment:
Are you sure about placing SEO tags outside of `<Head>`?

Reply:
According to README, It renders meta tags inside head tag. There is an example.

Reply:
https://github.com/garmeeh/next-seo#readme